docs/Routes.md

```diff
@@ -22,6 +22,7 @@ They need to be in
   * `params`: validates the params.
   * `response`: filter and generate a schema for the response, setting a
     schema allows us to have 10-20% more throughput.
+* `attachValidation`: attach validation result to request, if there is a schema validation error, instead of sending back the error immediately.
```

Comment:
I think this should read, `instead of sending the error to the error handler.`, since that's what actually happens when this option is `false`.